standard.go

```diff
@@ -0,0 +1,30 @@
+package producer
+
+import (
+	"fmt"
+	"log"
+	"strings"
+)
+
+// StandardLogger implements the Logger interface using standard library loggers
+type StandardLogger struct {
```

Comment:
`Std` is better name imo.
```suggestion
type StdLogger struct {
```